public/index.html

```diff
@@ -11,12 +11,12 @@
       http-equiv="Content-Security-Policy"
       content="
       default-src *;
-      script-src 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.lr-ingest.io https://kit.fontawesome.com https://js.stripe.com https://unpkg.com/graphiql/graphiql.min.js https://unpkg.com/react-dom/umd/react-dom.production.min.js https://unpkg.com/react/umd/react.production.min.js;
+      script-src 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.lr-ingest.io https://js.stripe.com;
```

Comment:
We can probably re-evaluate more of these values but let's save that for a later PR.